test/src/comps/test_determiter.cc

```diff
@@ -1,8 +1,9 @@
-#include <blas.hh>
-#include <RandBLAS.hh>
-#include <RandLAPACK.hh>
-#include <gtest/gtest.h>
+#include "RandLAPACK.hh"
+#include "RandBLAS.hh"
```

Comment:
Technically, isn't RandBLAS an external project, in the sense that it should be included with brackets when used in RandLAPACK?

Reply:
lol, OK

Reply:
done